src/index.ts

```diff
@@ -166,9 +166,6 @@ function _decode(input: Buffer): Decoded {
       throw new Error('invalid RLP: expected string length to be greater than 55')
     }
     data = safeSlice(input, llength, length + llength)
-    if (data.length < length) {
-      throw new Error('invalid RLP: not enough bytes for string')
```

Comment:
This can be removed as this exact same check is actually done in `safeSlice`, so `safeSlice` would always throw if `data.length < length` condition is true, and thus this line cannot be reached.